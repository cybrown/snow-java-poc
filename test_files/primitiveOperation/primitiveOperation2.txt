#add(a, #mul(4, e) )