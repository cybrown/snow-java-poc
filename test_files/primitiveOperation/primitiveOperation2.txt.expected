<PrimitiveOperation>
    <Identifier>add</Identifier>
    <Identifier>a</Identifier>
    <PrimitiveOperation>
        <Identifier>mul</Identifier>
        <LiteralInteger>4</LiteralInteger>
        <Identifier>e</Identifier>
    </PrimitiveOperation>
</PrimitiveOperation>
